vendor/k8s.io/gengo/examples/go-to-protobuf/protobuf/cmd.go

```diff
@@ -78,7 +78,6 @@ func New() *Generator {
 			`k8s.io/kubernetes/federation/apis/federation/v1beta1`,
 			`k8s.io/kubernetes/pkg/apis/certificates/v1alpha1`,
 			`k8s.io/kubernetes/pkg/apis/imagepolicy/v1alpha1`,
-			`k8s.io/kubernetes/pkg/apis/storage/v1beta1`,
```

Comment:
Uh, I didn't notice this until I made this PR. @smarterclayton do these defaults really matter?